src/products/mozilla_org/testing.md

```diff
@@ -52,28 +52,62 @@ You can make linguistic changes directly in [Pontoon](https://pontoon.mozilla.or
 * Test the page layout on the leading locally developed browser if available.
 * Test the page layout on mobile devices of major platforms.
 
-## When can I see the localized page on the production server?
+## Page activation
 
-Updated translations are pushed to the production server almost daily.
+Updated translations are pushed to the production server regularly throughout the day.
 
-When a brand new page is available for localization, it won’t be enabled in production until it’s fully localized. When existing pages receive updates with new strings, this new content won’t be displayed on production until localized, to avoid displaying a mix of English and localized text.
-
-In some cases, pages receive major updates that require a complete rewrite of the template. If this happens, the old template is kept online only for a short period of time and, when removed, it will cause the URL to redirect users to the English version.
+* When a brand new page is available for localization, it won’t be enabled on production until it’s at least 80% complete. This means, after it reaches this threshold, the page will be on production in English and target language.
+* When an existing page receives updates with new strings, these new strings may or may not have fallback strings. For a page that is completely translated, depending on the rule set for the page, the update may appear once fully localized, or the update will be in English.
+* An activated page falling far behind with multiple updates will create bad user experience. The page will risk being disabled but deactivation will be done manually.
```

Comment:
I'd rephrase the second part along the line of "we might decide to disable it manually". It's not a risk, it's a decision you'd make.